go.mod

```diff
@@ -1,14 +1,20 @@
-module github.com/markbates/goth
+module github.com/damonkeys/goth
```

Comment:
Please revert this line

Reply:
Oops. Sorry. Should be fixed with 59a4506cdec732fef480e4bac5a34a3550836cdb